spec/features/firstVisit.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('', function() {
+
+  beforeEach(function() {
+    browser.get('http://localhost:3000/');
+  });
+
+  it('Should have a title', function() {
+    expect(browser.getTitle()).toEqual('Things ToDo');
+  });
+
+  it('Should allow a new item to be added', function() {
+
+
+
```

Comment:
You've already written the code allowing something to be added, but with no test!